src/test/java/com/foundation/trello/step/ManagerStepDef.java

```diff
@@ -83,6 +84,7 @@ public void getStatusCodeAsResponse(int statusCode) {
      */
     @And("I verify the response schema whit {string}")
     public void iGetASchema(String schema) {
-        SchemaValidator.validator(response, "schema/" + schema + "Schema.json");
+        boolean validator = SchemaValidator.validator(response, "schema/" + schema + "Schema.json");
```

Comment:
The concat ("schema/" + schema + "Schema.json") of this string bring to validator method please

Reply:
I fixed it.